AdyenCard/Components/Card/CoBadgedAnalyticsConfiguration.swift

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+@_spi(AdyenInternal) import Adyen
+
+internal struct CoBadgedAnalyticsConfiguration: AnalyticsStringDictionaryConvertible {
+
+    private let dualBrands: String?
```

Comment:
Public property name `brands` is different from private `dualBrands` creating some confusion, let's align?